From c897096d5bac2bab228779a2c5602fc4bd296a0a Mon Sep 17 00:00:00 2001
From: Antoine Martin <dev@ayakael.net>
Date: Sun, 2 Jan 2022 21:59:58 +0000
Subject: [PATCH 1/2] use version 5.0.4 of
 MicrosoftNETCorePlatformsPackageVersion for up-to-date RIDs

In stock build, an old version of MicrosoftNETCorePlatformsPackage is used.
Thus, dotnet5-runtime is shipped without RIDs for alpine 3.13 and up, causing
a segmentation fault for dotnet projects. Related to dotnet/runtime#24189 and
dotnet/runtime#50739. This patches eng/Versions.props to use latest version.

---
 eng/Versions.props | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/eng/Versions.props b/eng/Versions.props
index 89c46cbb735..8ddd5d17900 100644
--- a/eng/Versions.props
+++ b/eng/Versions.props
@@ -66,7 +66,7 @@
     <!-- CoreClr dependencies -->
     <MicrosoftNETCoreILAsmVersion>5.0.0-preview.8.20359.4</MicrosoftNETCoreILAsmVersion>
     <!-- Libraries dependencies -->
-    <MicrosoftNETCorePlatformsPackageVersion>5.0.0</MicrosoftNETCorePlatformsPackageVersion>
+    <MicrosoftNETCorePlatformsPackageVersion>5.0.4</MicrosoftNETCorePlatformsPackageVersion>
     <MicrosoftNETCoreTargetsPackageVersion>5.0.0</MicrosoftNETCoreTargetsPackageVersion>
     <SystemBuffersVersion>4.5.1</SystemBuffersVersion>
     <SystemCollectionsVersion>4.3.0</SystemCollectionsVersion>
-- 
2.34.1

